developers/weaviate/config-refs/datatypes.md

```diff
@@ -67,6 +67,47 @@ In the above example, our objects can be linked to:
 * **2** Articles and **5** Blogs
 * etc.
 
+## DataType: `object`
+
+:::info Available from version `v1.22`
+:::
+
+The `object` type allows you to store nested data structures in Weaviate. The data structure is a JSON object, and can be nested to any depth.
+
+For example, a `Person` class could have an `address` property, as an object. It could in turn include nested properties such as `street` and `city`:
```

Comment:
Hmm. I think in this case, `address` would remain be property, and `street` or `city` would be nested properties.

Reply:
Agreed, 'address' is a property. I should have written:

In turn, the 'address' property can have nested properties
or
It could, in turn, include nested properties